h2o-core/src/test/java/water/runner/KeysMRTask.java

```diff
@@ -0,0 +1,23 @@
+package water.runner;
+
+import org.junit.Ignore;
+import water.Job;
+import water.Key;
+import water.MRTask;
+import water.Value;
+
+@Ignore
+public abstract class KeysMRTask<T extends KeysMRTask<T>> extends MRTask<T> {
```

Comment:
is this class really necessary? seems like it has only one child

Reply:
Thanks. Removed.